src/UseCases/ChangeAddress/ChangeAddressRequest.php

```diff
@@ -162,4 +162,11 @@ public function setDonationReceipt( bool $donationReceipt ): self {
 		return $this;
 	}
 
+	public function isOnlyOptInDonationReceiptRequest(): bool {
+		if ( $this->address == '' ) {
```

Comment:
This class has no property named "address". As I've written in the use cas comment, You probably don't need an `if` check.

Reply:
Uhm, there is `$address`. This is not a getter, this is to check if we are in the case when only the opt in preference has been changed and the address form is empty.